TwitchLib.Api.Helix/Streams.cs

```diff
@@ -97,6 +97,37 @@ public Task<GetStreamsMetadataResponse> GetStreamsMetadataAsync(string after = n
 
             return TwitchGetGenericAsync<GetStreamsMetadataResponse>("/streams/metadata", ApiVersion.Helix, getParams);
         }
+
+        public Task<GetStreamTagsResponse> GetStreamTagsAsync(string broadcasterId, string accessToken = null)
+        {
+            var getParams = new List<KeyValuePair<string, string>>();
+            getParams.Add(new KeyValuePair<string, string>("broadcaster_id", broadcasterId));
+
+            return TwitchGetGenericAsync<GetStreamTagsResponse>("/streams/tags", ApiVersion.Helix, getParams, accessToken);
+        }
+
+        public Task ReplaceStreamTags(string broadcasterId, List<string> tagIds = null, string accessToken = null)
+        {
+            DynamicScopeValidation(AuthScopes.Helix_User_Edit_Broadcast, accessToken);
+
+            var getParams = new List<KeyValuePair<string, string>>();
+            getParams.Add(new KeyValuePair<string, string>("broadcaster_id", broadcasterId));
```

Comment:
broadcasterId is used here without checking if the broadcasterId is null or empty. Might want to add that security check. Same for the methods above this one.

Reply:
done